message.go

```diff
@@ -52,11 +52,19 @@ func IsRedisErr(err error) (ret *RedisError, ok bool) {
 // RedisError is an error response or a nil message from redis instance
 type RedisError RedisMessage
 
+// string retrives the contained string of the RedisMessage
+func (m *RedisError) string() string {
```

Comment:
```suggestion
func (m RedisError) string() string {
```
Is there any performance downside to using a value receiver on this function? Can this function be inlined?

Reply:
Both of value receiver and pointer receiver could be inlined.
Pointer receiver:
```
./message.go:56:6: can inline (*RedisError).string with cost 13 as: method(*RedisError) func() string { if m.bytes == nil { return "" }; return <node unsafe.String> }
```

Value receiver:
```
./message.go:56:6: can inline RedisError.string with cost 13 as: method(RedisError) func() string { if m.bytes == nil { return "" }; return <node unsafe.String> }
```

Additionally, it seems have around 25% improve on benchmarking for a pointer receiver. Attached the benchmark code for reference.
```
BenchmarkValueReceiver
BenchmarkValueReceiver-4     	493528951	         2.412 ns/op	       0 B/op	       0 allocs/op
BenchmarkPointerReceiver
BenchmarkPointerReceiver-4   	690465366	         1.777 ns/op	       0 B/op	       0 allocs/op
```
[message_bench_test.go.txt](https://github.com/user-attachments/files/19407155/message_bench_test.go.txt)